app/models/audio_file.rb

```diff
@@ -1,6 +1,17 @@
 class AudioFile < ApplicationRecord
   belongs_to :user
+  has_many :activities
```

Comment:
In addition to adjusting from `has_many` you probably need a `dependent: :destroy` option here. That way, if the audio file is deleted, the associated activity will also be removed to avoid orphan records.